core/stores/redis/redis.go

```diff
@@ -765,6 +795,21 @@ func (s *Redis) Llen(key string) (val int, err error) {
 	return
 }
 
+// Lindex is the implementation of redis lindex command.
+func (s *Redis) Lindex(key string, index int64) (val string, err error) {
```

Comment:
please add unit test for this func.

Reply:
sorry  I was a little careless
i've my check unit test and sort func